README.md

```diff
@@ -343,27 +341,28 @@ You can use the following arguments:
 For example:
 
 ```bash
-node dist/cli.js setup polkadot polkadot-parachain
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup polkadot polkadot-parachain
 ```
 
-> Note: If you are using macOS please clone the [Polkadot repo](https://github.com/paritytech/polkadot) and run it locally. At the moment there is no `polkadot` binary for MacOs.
+> Note: If you are using macOS please clone the [polkadot-sdk repo](https://github.com/paritytech/polkadot-sdk) and run it locally. At the moment there is no `polkadot` binary for MacOs.
 
 The command above will retrieve the binaries provided and try to download and prepare those binaries for usage. 
 At the end of the download, the `setup` script will provide a command to run in your local environment in order to add the directory where the binaries were downloaded in your $PATH var, for example:
 
 ```bash
-Please add the dir to your $PATH by running the command: export PATH=/home/<user>/zombienet/dist:$PATH
+Please add the dir to your $PATH by running the command: export PATH=/<full path to the project>/zombienet/dist:$PATH
```

Comment:
It is possible that repo was cloned in a directory different from `/home/<user>`

Reply:
I really think that this is not needed to be so specific... but sure..

Reply:
This path is old `/zombienet/dist:$PATH` is we keep this comment we should update it.

Reply:
returned previous path back